searchlib/src/vespa/searchlib/attribute/integerbase.cpp

```diff
@@ -49,18 +59,19 @@ uint32_t IntegerAttribute::get(DocId doc, WeightedConstChar * v, uint32_t sz) co
 }
 const char *
 IntegerAttribute::getString(DocId doc, char * s, size_t sz) const {
-    largeint_t v = getInt(doc);
-    snprintf(s, sz, "%" PRId64, v);
+    if (sz > 1) {
+        largeint_t v = getInt(doc);
+        auto res = std::to_chars(s, s + sz - 1, v, 10);
+        res.ptr[0] = 0;
```

Comment:
consider checking ec to avoid returning unspecified results.

Reply:
Fixed